router/handlers_test.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+var nopHandler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {})
+
+func TestFwdProtoHandler(t *testing.T) {
```

Comment:
yeah not sure. We had these tests already from one of our own project so I just imported them as-is (with additions for `X-Forwarded-For`). Let me know if you want to switch this to using gocheck instead.